### PR TITLE
Salt Day 7 Part 1

### PR DESCRIPTION
**Pull request by Parzivus**
Includes everything through 7-4. Volvo has already finished the full day so we can keep going on the group PR if we get super far.

### CODE REVIEW

File: script/Ciel/Day 7/07_01_CIEL07_1.txt
Comment (Avuxo) [inline script/Ciel/Day 7/07_01_CIEL07_1.txt:379]: revert

File: script/Ciel/Day 7/07_01_CIEL07_1B.txt
Comment (Avuxo) [inline script/Ciel/Day 7/07_01_CIEL07_1B.txt:150]: revert

File: script/Ciel/Day 7/07_01_CIEL07_1B.txt
Comment (Avuxo) [inline script/Ciel/Day 7/07_01_CIEL07_1B.txt:260]: revert

File: script/Ciel/Day 7/07_01_CIEL07_1_1.txt
Comment (Avuxo) [inline script/Ciel/Day 7/07_01_CIEL07_1_1.txt:36]: revert or i guess it could be "lads' lunch" but thats a little bit ridiculous

File: script/Ciel/Day 7/07_01_CIEL07_2_1MERGE.txt
Comment (Avuxo) [inline script/Ciel/Day 7/07_01_CIEL07_2_1MERGE.txt:20]: revert or convert to 2x cjk. one dash is just weird

File: script/Ciel/Day 7/07_01_CIEL07_2_2.txt
Comment (Avuxo) [inline script/Ciel/Day 7/07_01_CIEL07_2_2.txt:479]: revert

**Review by Avuxo — APPROVED** [review]

File: script/Ciel/Day 7/07_01_CIEL07_2_2.txt
Comment (Petrikow) [inline script/Ciel/Day 7/07_01_CIEL07_2_2.txt:98]: Noel calls him a lamb > noel is 'attacking' him > shiki likes it > maybe 'being a lamb = living in the pastures' isn't that bad?

**Review by rschlaikjer — COMMENTED** [review]
lgtm pending Ben/Petrikow comments

File: script/Ciel/Day 7/07_01_CIEL07_1.txt
Comment (Parzivus) [reply script/Ciel/Day 7/07_01_CIEL07_1.txt:379]: I thought we were getting rid of all of these?

File: script/Ciel/Day 7/07_01_CIEL07_1B.txt
Comment (Parzivus) [reply script/Ciel/Day 7/07_01_CIEL07_1B.txt:150]: This one definitely doesn't need a comma. Suppose it could be "I'm both surprised and overjoyed to run into Senpai here."

**Comment by Parzivus** [comment]
Gonna set my alarm for like 30 minutes before the GPR for last minute revisions and merge before we start, one of you can get it if you want to download everything sooner

**Review by yk1512 — APPROVED** [review]

**Pull request merged**
